nomad/grapl-core.nomad

```diff
@@ -236,8 +236,27 @@ job "grapl-core" {
 
       env {
         # Upstreams
-        PLUGIN_WORK_QUEUE_CLIENT_ADDRESS = "http://${NOMAD_UPSTREAM_ADDR_plugin-work-queue}"
-        PLUGIN_REGISTRY_CLIENT_ADDRESS   = "http://${NOMAD_UPSTREAM_ADDR_plugin-registry}"
+        PLUGIN_WORK_QUEUE_CLIENT_ADDRESS                       = "http://${NOMAD_UPSTREAM_ADDR_plugin-work-queue}"
+        PLUGIN_WORK_QUEUE_CLIENT_REQUEST_TIMEOUT               = "1s"
```

Comment:
i sort of don't love all of these being in env{} - perhaps an eventual config file instead, since figment supports all sorts of difference sources - **but this is totally fine to land for now**